src/index.js

```diff
@@ -0,0 +1,66 @@
+let temp=83;
+
+const changeTempColor = (currentTemp)=> {
+    const tempValueEl = document.getElementById("tempValue");
+    let color = "teal";
+    if (currentTemp>=80) {
+        color='orange';
+    } else if (currentTemp>=70) {
+        color='yellow';
+    }else if (currentTemp>=60) {
+        color='green';
+    } else if (currentTemp>=50) {
+        color='blue';
+    }
+    tempValueEl.classList = color;
+}
+
+const updateTemp = () => {
+    const tempValueEl = document.getElementById("tempValue");
+    tempValueEl.textContent=temp;
+    changeTempColor(temp);
+}
+
+const clickIncreaseTemp=()=> {
+    const tempEl=document.getElementById('tempValue')
+    temp+=1;
+    tempEl.textContent = temp
```

Comment:
after you increase the temperature you want to call your updateTemp function
```suggestion
    updateTemp(temp)
```